Mathlib/NumberTheory/ModularForms/SlashInvariantForms.lean

```diff
@@ -208,9 +208,14 @@ theorem coeHom_injective : Function.Injective (@coeHom Γ k) :=
 instance : Module ℂ (SlashInvariantForm Γ k) :=
   coeHom_injective.module ℂ coeHom fun _ _ => rfl
 
-instance : One (SlashInvariantForm Γ 0) :=
-  ⟨{toFun := 1
-    slash_action_eq' := fun A => ModularForm.is_invariant_one A }⟩
+/-- The `SlashInvariantForm` corresponding to `Function.const _ x`. -/
+@[simps]
+def const (x : ℂ) : SlashInvariantForm Γ 0 where
+  toFun := Function.const _ x
+  slash_action_eq' A := ModularForm.is_invariant_const A x
```

Comment:
@CBirkbeck, I just pushed this; can you add an analogous definition for `ModularForm`?

Reply:
(this will also be useful for building a `GAlgebra` instance, I would guess)

Reply:
I went ahead and did this for you :)

Reply:
Oh thank you! it looks a lot cleaner now :)